### ISSUE TITLE
VAB is broken after reverting flight with a KAS winch

### ISSUE THREAD

**Issue by ihsoft** [ihsoft/KAS#272]
Add any winch part to a vessel and go to the launchpad. Then, revert back to VAB. There are exceptions in the log and no parts can be grabbed in the editor.

```
190604T191048.524 [EXCEPTION] [ShipConstruct.LoadShip] NullReferenceException: Object reference not set to an instance of an object
   at ShipConstruct.LoadShip (.ConfigNode root, UInt32 persistentID, Boolean returnErrors, System.String& errorString)
   at ShipConstruct.LoadShip (.ConfigNode root, UInt32 persistentID)
   at ShipConstruct.LoadShip (.ConfigNode root)
   at ShipConstruction.LoadShip ()
   at EditorLogic.StartEditor (Boolean isRestart)
   at EditorLogic+<Start>c__Iterator0.MoveNext ()
   at UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress)
```

**Comment by ihsoft** [comment]
This issue has to be fixed in the game's core. Let's wait for the next version.

**Comment by lobf** [comment]
Also having this issue, hope to see it resolved soon.

**Comment by ihsoft** [comment]
It's fixed in `KSP 1.7.2`
